### ISSUE TITLE
[RotaryCraft] Fluid Overlay in Pipes Don't Match Up to Pipe Model

### ISSUE THREAD

**Issue by developer9199** [ReikaKalseki/Reika_Mods_Issues#2643]
When a RotaryCraft or ReactorCraft fluid enters any RotaryCraft pipe (e.g., bedrock pipe, liquid pipe, e.t.c.) the overlay that's created doesn't line up with the model. Specifically at joints where multiple pipes meet up. I noticed this while building a Tokamak reactor. Interestingly, I don't see this issue when for example, fusion plasma runs through magnetic containment pipes.

I have attatched screenshots of the issue and a .txt containting the list of mods that I used in the world, with the hopes that a specific mod could be causing an issue with certain rendering techniques used to create the pipes. Or it could be a simple bug.

Forge 1.7.10-10.13.4.1614
macOS 10.15 (Java v12.0.2+10)

[mods_used.txt](https://github.com/ReikaKalseki/Reika_Mods_Issues/files/5222425/mods_used.txt)

![2020-09-14_19 59 20](https://user-images.githubusercontent.com/61306129/93164721-18070d80-f6cf-11ea-891e-c4a8b666f287.png)

![2020-09-14_19 59 29](https://user-images.githubusercontent.com/61306129/93164728-1ccbc180-f6cf-11ea-84c5-6a7fa2c0f45a.png)

**Comment by ReikaKalseki** [comment]
This has happened occasionally since 1.6. I still have no idea why.

**Comment by developer9199** [comment]
Dang. Do you happen to know if there are any work-arounds? I assume that if you have no idea why it happens then you probably won't know how to fix it, but you never know. Also, I appreciate the timely response. It's awesome to see developers that still maintain and debug mods on relatively old versions.

**Comment by ReikaKalseki** [comment]
It in fact happens as a result of a workaround, namely that I flip the renderer into intentionally the *wrong* direction to fix it rendering incorrectly normally.

**Comment by MalTeeez** [comment]
I have this happening a lot when loads of blockupdates happen around the pipes, mostly related to steam - is that related?

**Comment by ReikaKalseki** [comment]
No, that is a different issue that only affects TESRs (or rather, the client-sided TileEntities, and only TESRs respond fast enough to visibly change).
